src/v3/src/util/flattenInputs.ts

```diff
@@ -12,19 +12,34 @@
 
 import { Input } from '@okta/okta-auth-js';
 
-export const flattenInputs = (input: Input, name = ''): Input[] => {
-  const res: Input[] = [];
-  const { value } = input;
+export const flattenInputs = (input: Input): Input[] => {
```

Comment:
The comments are great, it would be nice if there was a unit test for this that displayed the various scenarios (calling flattenInputs on credentials object, on identifier field, on fields that are mutable but not visible, etc).